examples/configurations/api/anthropic_api.py

```diff
@@ -0,0 +1,19 @@
+import os
+from llama_index.llms.anthropic import Anthropic
+
+
+class AnthropicApiLLM(Anthropic):
```

Comment:
We need to call the class 'LLM' as LaVague expects any config LLMs provided to be called `LLM` - if not, it just uses the default OpenAI